segment_executiontime.go

```diff
@@ -35,3 +41,115 @@ func (t *executiontime) init(props *properties, env environmentInfo) {
 	t.props = props
 	t.env = env
 }
+
+func (t *executiontime) formatDuration(ms int64, format string) string {
+	switch format {
+	case "austin":
```

Comment:
In other segments (path for example), these are created as a separate type (maybe `DurationFormat`).

Reply:
Just making sure I'm on the same page. I'm looking at `segment_path.go`. You're asking for constants, right?
Also, unrelated, I see path calls a similar attribute `style` instead of `format`. I'm thinking executionTime should also go with the name `style`, for consistency.

Reply:
About style, YES!

Reply:
@JanDeDobbeleer  all done!